doc/api/fs.md

```diff
@@ -726,6 +731,54 @@ added: v8.1.0
 The timestamp indicating the creation time of this file expressed in
 milliseconds since the POSIX Epoch.
 
+### stats.atimeNs
+<!-- YAML
+added: REPLACEME
```

Comment:
This should be changed, right?

Reply:
That'll only be available when this is released.